trashtalk/seeclickfix.py

```diff
@@ -1,5 +1,6 @@
 import json
 
+
```

Comment:
Remove.

Reply:
My machine insists I explicitly import it, otherwise it malfunctions. I really don't want to pick a fight with it. I'll just comment out the pushed version.